appdirs.py

```diff
@@ -41,6 +41,12 @@
     system = sys.platform
 
 
+def getenv(name, default=None):
+    value = os.getenv(name)
+    if not value:
+        value = default
+    return value
```

Comment:
Wouldn't the entire function be able to be implemented using solely `return os.getenv(name) or default`?

Reply:
Yep! I'd still prefer it in a function but the shorter version would be fine.